src/components/EditPicture.tsx

```diff
@@ -161,6 +163,51 @@ export const EditPicture = ({
     }
   };
 
+  const handlePickImage = async () => {
+    try {
+      let permissionResult = await ImagePicker.requestMediaLibraryPermissionsAsync();
+      if (permissionResult.granted === false) {
+        openModal("Error", { msg: t`Permission to access photo album is required` });
+        return;
+      }
+
+      let result = await ImagePicker.launchImageLibraryAsync({
+        allowsEditing: true,
+        aspect: [1, 1],
+        quality: 0.5,
+        exif: false,
+      });
+
+      if (!result.canceled) {
+        setLoading(true);
+        if (pic) {
+          const hash = pic.split("/ipfs/")?.[1] || undefined;
+          await removePinFromIPFS(hash)
```

Comment:
Unfortunately there is no easy way to verify that the CID being deleted was indeed uploaded by the same user trying to delete it. So it's unsafe to allow something like this and probably better to remove this

Reply:
Sure... I forgot that the pic string is used in TextInput, which might be accidentally unpinned wrong CID.